### ISSUE TITLE
Clean up indenting

### ISSUE THREAD

**Issue by sammefford** [marklogic/java-client-api#687]
It's time to make the indenting consistent across the Java Client API

**Comment by jmakeig** [comment]
Do we have a shared code style that everyone could use (and CI could potentially enforce)? I use https://gist.github.com/jmakeig/cf8d297efe13bb3d7fb0, but I haven’t given it much thought.

**Comment by sammefford** [comment]
Sorry, @jmakeig, the code style from Eclipse and IntelliJ Idea covers way more than indenting, and would have changed the code more than I was comfortable.  So I've just committed the indenting changes.  The main rule is 2 space indent so less code will need to wrap because it's too far to the right, and no tabs.

**Comment by sammefford** [comment]
I'll make a separate task to clean up other formatting and code style things besides just indenting.

**Comment by sammefford** [comment]
Here's the IntelliJ formatting rules I used together with CTRL-ALT-I to indent the files.  After IntelliJ indenting, I made a couple changes by hand:

1. re-indent wrapped "implements" classes that IntelliJ wanted to align
2. move open brace for wrapped method declarations to next line

[IntelliJ_Idea_formatting.zip](https://github.com/marklogic/java-client-api/files/869342/IntelliJ_Idea_formatting.zip)
